Makefile

```diff
@@ -52,6 +52,11 @@ prow-config:
 	docker pull registry.svc.ci.openshift.org/ci/determinize-prow-config:latest
 	docker run -it -v "${CURDIR}/core-services/prow/02_config:/config" registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest --prow-config-dir /config
 
+branch-cut:
+	docker pull registry.svc.ci.openshift.org/ci/config-brancher:latest
+	docker run -it -v "${CURDIR}/ci-operator:/ci-operator" registry.svc.ci.openshift.org/ci/config-brancher:latest --config-dir /ci-operator/config --org=$(ORG) --repo=$(REPO) --current-release=4.3 --future-release=4.4 --bump-release=4.4 --confirm
```

Comment:
should these release #s be parameters also?

Reply:
No. Changing what the release is is a cut over task. I want to give people the option to cut over within the boundary of the current release.